rules/combo/stealer/crypto.yara

```diff
@@ -11,7 +11,9 @@ rule js_crypto_stealer : high {
     $url = /https{0,1}:\/\/[\w][\w\.\/\-_\?=\@]{8,64}/
 
 	$POST = "POST"
-
+	$not_webdav = "WebDAV certificate"
+	$not_letsencrypt = "Letsencrypt"
```

Comment:
Would `/[Ll]etsencrypt/` work here?

Reply:
It would, but it will would execute a tiny bit more slowly. If we end up with a 3rd option, then I'd combine it into a regexp.